web/js/map-OpenLayers.js

```diff
@@ -90,7 +90,8 @@ var fixmystreet = fixmystreet || {};
                 size: pin[5] || marker_size,
                 faded: 0,
                 id: pin[3],
-                title: pin[4] || ''
+                title: pin[4] || '',
+                draggable: pin[6] === false ? false : true
```

Comment:
This sets draggable to be true on all markers e.g. on around pages? I realise the DragFeature isn't used, but it might be better to default to false?